integration_test/third_party_apps_data/applications/zookeeper/metadata.yaml

```diff
@@ -22,6 +22,9 @@ description: |-
   latency, active requests, and active connections. The integration also collects
   ZooKeeper logs and parses them into a JSON payload. The result includes fields
   for node ID, source, level, and message.
+configure_integration: |-
+  The Zookeeper receiver collects metrics from a Zookeeper instance, using the `mntr` command.
+  The `mntr` 4 letter word command needs to be enabled for the receiver to be able to collect metrics.
```

Comment:
Not sure what "4 letter word command" means here. Is this something other than "mntr"? If not, I'd just say "The \'mntr\' command must be enabled for the receiver to be able to collect metrics."

Do we need to tell people how to "enable" this command -- I'm not sure what that means, either.